lib/membrane_mp4/muxer/cmaf.ex

```diff
@@ -359,6 +359,16 @@ defmodule Membrane.MP4.Muxer.CMAF do
   defp generate_output_stream_format(state) do
     tracks = Enum.map(state.pad_to_track_data, fn {_pad, track_data} -> track_data.track end)
 
+    resolution =
+      tracks
+      |> Enum.map(fn
+        %Track{width: 0} -> nil
+        track -> {track.width, track.height}
+      end)
+      |> Enum.find(&(not is_nil(&1)))
+
+    codecs = Map.new(tracks, fn track -> Track.get_encoding_info(track) end)
```

Comment:
Can we generate that in respective payloaders?

Reply:
Maybe we could, but this would require extending `Membrane.MP4.Payload`, and I am not sure if this makes sense as this added information will not be used anywhere besides HLS.

Reply:
I think it would be used only in the CMAF muxer, which constitutes half of the use cases :D Anyway, at least make use of the functions from the aac format to generate the codec info.